src/cycax/cycad/engines/part_freecad.py

```diff
@@ -19,11 +19,15 @@ def build(self) -> dict:
             logging.error("Use freeCAD %s", app_bin)
             freecad_py = Path(sys.modules[self.__module__].__file__).parent / "cycax_part_freecad.py"
 
+            config_csv = self.config['out_formats']
```

Comment:
Do you really trust users? Humans are incredibly unreliable.

The CSV should be built up by code.

Maybe:
```
out_formats_set = set()
for file_format in self.confg.get('out_formats', []):
     out_formats_set.add(":".join(file_format[:2]))
 
environment.update({"CYCAX_OUT_FORMATS": ",".join(out_formats_set)})
```